service/src/main/java/com/epam/esm/service/exeption/IllegalRequestParameterException.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.esm.service.exeption;
+
+public class IllegalRequestParameterException extends Exception{
+    public IllegalRequestParameterException(String message) {
```

Comment:
all the custom exceptions should have constructors that accepts `Throwable cause` as well

Reply:
done